crypto-ffi/src/generic.rs

```diff
@@ -945,8 +973,8 @@ impl CoreCrypto {
     pub fn set_logger(&self, logger: std::sync::Arc<dyn CoreCryptoLogger>, level: CoreCryptoLogLevel) {
```

Comment:
Is the logger is global and not bound to specific core crypto instance? Because me as a client call this on a core crypto instance but it will I think saw that it throws an error if try to do it again on a second core crypto instance.

Reply:
yes it is global

Reply:
Can't it be a static function then?